cms_helper.py

```diff
@@ -25,7 +25,7 @@ def gettext(s): return s  # NOQA
     LANGUAGE_CODE='en',
     LANGUAGES=(
         ('en', gettext('English')),
-        ('fr_FR', gettext('French')),
+        ('fr-fr', gettext('French')),
```

Comment:
@yakky Why this is not only 'fr' ?

Reply:
Right I think it's the french spoken only in France (fr-FR)

Reply:
Yes, "fr" is the language, but "fr-fr" is the locale (which brings also number formatting, currency etc into the configuration) and "fr-ca" is different from "fr-fr" (for example currency symbol is $ instead of €)

Reply:
:+1: thanks!